buildscripts/Build-NetCore.ps1

```diff
@@ -0,0 +1,34 @@
+$dnxHome = Join-Path (Get-Location) ".dnx"
+$dnxPath = Join-Path $dnxHome "bin"
+
+Write-Host "Downloading dnvm to: $dnxPath"
+if (!(Test-Path $dnxPath)) { md $dnxPath | Out-Null }
+
+$dnvmPs1Path = Join-Path $dnxPath "dnvm.ps1"
+$dnvmCmdPath = Join-Path $dnxPath "dnvm.cmd"
+
+$wc = New-Object System.Net.WebClient
+$wc.DownloadFile('https://raw.githubusercontent.com/aspnet/Home/dev/dnvm.ps1', $dnvmPs1Path)
+$wc.DownloadFile('https://raw.githubusercontent.com/aspnet/Home/dev/dnvm.cmd', $dnvmCmdPath)
+
+Write-Host "Downloading dnx"
+
+$env:PATH = ("$dnxPath;" + $env:PATH)
+
+dnvm upgrade -r coreclr
+
+dnx --version
+
+Write-Host "Downloading packages"
+
+dnu restore
+
+Write-Host "Building"
+
+$env:Path = ((Join-Path $dnxHome "runtimes\dnx-coreclr-win-x64.1.0.0-rc1-update1\bin") + ";" + $env:Path)
+
+dnu build src/Castle.Core src/Castle.Core.Tests --configuration Release --out build/NETCORE
```

Comment:
I know this is what I already had, but does DNX have a feature to build using the global.json file so we don't have to add each logging service here too?